src/test/java/api/support/matchers/TextDateTimeMatcher.java

```diff
@@ -46,13 +48,14 @@ public void describeTo(Description description) {
       @Override
       protected boolean matchesSafely(String textRepresentation) {
         //response representation might vary from request representation
-        Instant actual = ZonedDateTime.parse(textRepresentation).toInstant();
+        final var actual = OffsetDateTime.parse(textRepresentation);
 
         //The zoned date time could have a higher precision than milliseconds
         //This makes comparison to an ISO formatted date time using milliseconds
         //excessively precise and brittle
         //Discovered when using JDK 13.0.1 instead of JDK 1.8.0_202-b08
-        return expected.truncatedTo(MILLIS).equals(actual);
+        return expected.truncatedTo(MILLIS).equals(actual.toInstant())
+          && actual.getOffset().equals(UTC);
```

Comment:
Can you add the time zone requirement to the describeTo string and to the isEquivalentTo javadocs?

`"a date and time in UTC zone (= 0 offset) matching: %s"`

`/** zone is UTC (offset 0) and the point in time is the same */`

Reply:
I've expanded that description to make that clearer. Please re-open this comment or comment again, if that does not work for you